CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Next Version
 
+#### Fixed
+- Improved variable expansion runtime and deprecated $old_form in favor of ${new_form} [#704](https://github.com/yonaskolb/XcodeGen/pull/704) @rcari
```

Comment:
Can we split this into 2 changelog items, with the deprecation going into a `Changed` section